sdk/canton/ref

```diff
@@ -1 +1 @@
-20240702.12121.v5da13c69-dirty
```

Comment:
This is the main change over Remy's PR, aside from the change to the refresh-canton.sh, yes?